### ISSUE TITLE
[OGRE21] SceneryManager needs to permit to change parameters of HDR rendering

### ISSUE THREAD

**Issue by Ybalrid** [Ybalrid/Annwvyn#94]
There's new things to cake care of with the HDR pipeline : 

- You always need some ambient lighting. If you don't have any, when the camera is looking at a light level of "0", it now don't just correspond to black pixels, it literally mean "No lights at all", that's only possible if your are looking at *the inside of a black hole.* This makes the camera crank up the exposure until the rendering *glitch out completely* (see screenshot below)

![test_2017-02-23_01-14-17](https://cloud.githubusercontent.com/assets/1816251/23265021/ab31d50a-f9e3-11e6-8bea-451267e2b304.png)

- The color of the sky, the camera exposure and some other parameters are needed to "set" how the scene should look. This role should be delegated to the SceneryManager too.

- I still have to check how it work, but some amount of Global Illumination is needed to get "realistic" lighting, because light bounce of surfaces. Ogre can do that, and the SceneryManager should be able 

So, in short, we have to do : 

 - [x]  Ambient Lighting
 - [x]  Exposure values
 - [x]  Sky Color
 - [x]  Bloom Threshold
<del> - [ ]  AO or GI or any way to get some light inside the shadows so that it look more "right"</del>

**Comment by Ybalrid** [comment]
Currently, the AnnSceneryManager  can set the exposure and the sky color. There's still the bloom threshold and the ambiant lighting to use.

I still don't know what I'm going to do about Ambiant Oclusion and/or Global Illumination.

**Comment by Ybalrid** [comment]
Added bloom

**Comment by Ybalrid** [comment]
I'm not sure about what to do concerning AO/GI ans other listing tricks. I have to study the ssao sample from Ogre to get an idea on how it works.

**Comment by Ybalrid** [comment]
We will not do AO for now, will close soon.
